package.json

```diff
@@ -562,6 +560,7 @@
     "@types/mime": "^2.0.1",
     "@types/mime-types": "^2.1.0",
     "@types/minimatch": "^2.0.29",
+    "@types/minimist": "^1.2.1",
```

Comment:
__note:__ before that we relied on the types that were used by the transitive dependencies.

Reply:
As far as I can see the transitive dependency uses `^1.2.0`, which will also match v1.2.1. So I think if we just delete it from `yarn.lock` and bootstrap again, we don't need this resolution, right?

Reply:
Well, it's not a resolution, it's a dev dependency, or what do you mean?

Reply:
Oh sorry, yeah I thought it was a resolution for some reason 🤦